src/Platforms/AbstractPlatform.php

```diff
@@ -556,10 +556,14 @@ public function getSumExpression(string $value) : string
     /**
      * Returns the SQL snippet to get the concatenation of the values in a set with a separator.
      *
-     * @param string $value     SQL expression producing the value.
-     * @param string $separator SQL expression producing the separator.
+     * @param string      $value     SQL expression producing the value.
+     * @param string      $separator SQL expression producing the separator.
+     * @param string|null $orderBy   Optional SQL expression (and direction) to order the grouped rows by.
      */
-    abstract public function getAggregateConcatExpression(string $value, string $separator) : string;
+    public function getAggregateConcatExpression(string $value, string $separator, ?string $orderBy) : string
```

Comment:
```suggestion
    public function getAggregateConcatExpression(string $value, string $separator, ?string $orderBy = null) : string
```

Reply:
Wow, you're almost as fast as PhpStan and Psalm! :)